### PR TITLE
Upgrade angular

### PR DESCRIPTION
**Pull request by Minglee01**
@grahamgilchrist

### CODE REVIEW

File: tests/inputFieldsSpec.js [hunk 1]
```diff
@@ -572,8 +572,8 @@
 
                     $rootScope.$digest();
 
-                    expect($(compiledTemplate, 'select[ng-model="hours"] option:nth-child(1)').attr('label')).toBe('00');
-                    expect($(compiledTemplate, 'select[ng-model="hours"] option:nth-child(10)').attr('label')).toBe('09');
+                    expect($(compiledTemplate, 'select[ng-model="hours"] option:nth-child(2)').attr('label')).toBe('00');
```
Comment (grahamgilchrist) [inline tests/inputFieldsSpec.js]: @Minglee01 Why have these tests changed?

**Comment by grahamgilchrist** [comment]
@Minglee01 FYI hopefully the last commit will fix the angular version. Annoyingly this resolution will also need to be in all projects which use this package, so we should maybe think about how to update or replace this slider in future...

File: tests/inputFieldsSpec.js (hunk 1 above)
Comment (Minglee01) [inline tests/inputFieldsSpec.js]: not sure they failed before i made the change

Comment (grahamgilchrist) [inline tests/inputFieldsSpec.js]: The tests are passing in master, so has something changed in one of the dependencies that causes this?

Comment (Minglee01) [inline tests/inputFieldsSpec.js]: possibly

**Pull request merged**